contracts/ClearingHouse.sol

```diff
@@ -1104,12 +1129,16 @@ contract ClearingHouse is
     }
 
     function _getTakerPositionSafe(address trader, address baseToken) internal view returns (int256) {
-        int256 takerPositionSize = IAccountBalance(_accountBalance).getTakerPositionSize(trader, baseToken);
+        int256 takerPositionSize = _getTakerPosition(trader, baseToken);
         // CH_PSZ: position size is zero
         require(takerPositionSize != 0, "CH_PSZ");
         return takerPositionSize;
     }
 
+    function _getTakerPosition(address trader, address baseToken) internal view returns (int256) {
```

Comment:
suggest renaming to `_getTakerPositionSize()`

Reply:
we want to follow the naming like `_getTakerPositionSafe` function, in this case, we don't want to update `_getTakerPositionSafe` to `_getTakerPositionSizeSafe`, so we would suggest keeping the naming same, what do you think?

Reply:
line 1131 > _getTakerPositionSafe

Reply:
ah ok, didn't realize there's already another function doing this. Fine then